docs/user-guide/content-sources.rst

```diff
@@ -98,6 +103,7 @@ The pulp-admin client can be use to list all defined content sources as follows:
   Base URL:       http://content-world/content/
   Enabled:        1
   Expires:        3d
+  Headers:        X-RHUI-ID=1234 \ X-Csrf-Token=31415926535897
   Max Concurrent: 2
   Name:           Content World
   Paths:          f18/x86_64/os/ f18/i386/os/ f19/x86_64/os f19/i386/os
```

Comment:
Actually, while you're at it (and I know you did not edit this part), I would appreciate the paths be on separate lines as well.